test/e2e/csv_e2e_test.go

```diff
@@ -148,7 +148,7 @@ func newNginxDeployment(name string) appsv1.DeploymentSpec {
 				Containers: []corev1.Container{
 					{
 						Name:  genName("nginx"),
-						Image: "bitnami/nginx:latest",
+						Image: "redis",
```

Comment:
🙂 
Note: It would be nice not to worry about this in the future. Maybe, in the future, we could use a standard base image configured to run a "generic test operator" script.